librosa/core/spectrum.py

```diff
@@ -1587,7 +1587,7 @@ def db_to_amplitude(S_db, ref=1.0):
 
 
 @cache(level=30)
-def perceptual_weighting(S, frequencies, **kwargs):
+def perceptual_weighting(S, frequencies, kind='A', **kwargs):
     '''Perceptual weighting of a power spectrogram:
 
     `S_p[f] = A_weighting(f) + 10*log(S[f] / ref)`
```

Comment:
Probably should update this part too

Reply:
hmm update how? I added `kind='A', kind='C'`. Idk am I missing something?

Reply:
:+1: thanks!

Reply:
I think this didn't highlight properly -- what needs updating is the docstring, since it's no longer always the A_weighting.